parallel-install/pkg/installation/installation_test.go

```diff
@@ -0,0 +1,93 @@
+package installation
+
+import (
+	"context"
+	"fmt"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/components"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/config"
+	"github.com/stretchr/testify/assert"
+	"log"
+	"testing"
+	"time"
+)
+
+const cancelTimeout = 50
+const quitTimeout = 150
+const cancelTimeoutMillisecond = time.Duration(50) * time.Millisecond
+const quitTimeoutMillisecond = time.Duration(150) * time.Millisecond
+
+
+func Test_CancelTimeout(t *testing.T) {
```

Comment:
Please provide a tests for entire flow (StartKymaInstallation)
- happy path
- cancel timeout in prerequisites (ensure normal install doesn't start)
- quit timeout in the middle of prerequisites installation (ensure it quits immediately even if some prerequisites are still being installed in the background)